src/components/hv-text/index.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
```

Comment:
I created [a task](https://app.asana.com/0/923014529014430/1154294366555337/f) if you feel strongly about this, so this can be applied to the entire codebase